src/styles/components/_Forms.scss

```diff
@@ -0,0 +1,62 @@
+.Form_Field,
+input[type="text"],
+input[type="search"],
+input[type="password"],
+input[type="email"] {
+	@include font-body;
+	@include text-std;
+	border: 1px solid $border-color;
+	padding: #{ $base-line-height / 2 } 1em;
+	background: transparent;
+	color: $hm-warm-grey;
+	margin-bottom: $base-line-height / 2;
+	margin-right: $base-line-height / 2;
+	display: block;
+	text-align: left;
+	border-radius: 2px;
+	vertical-align: middle;
+	width: 300px;
```

Comment:
Perhaps use percentage widths instead of `px` throughout, as this will be better responsively